soh/soh/ResourceManagerHelpers.cpp

```diff
@@ -11,9 +11,18 @@
 #include "resource/type/Array.h"
 #include "resource/type/Skeleton.h"
 #include "resource/type/PlayerAnimation.h"
+#include "soh/ActorDB.h"
+#include "soh/resource/type/Scene.h"
+#include "soh/resource/type/scenecommand/SetRoomList.h"
+#include "soh/resource/type/scenecommand/SetActorList.h"
+#include "soh/resource/type/scenecommand/SetObjectList.h"
 #include <Fast3D/gfx_pc.h>
 #include <DisplayList.h>
 
+bool delayedAltAssetLoad = false;
```

Comment:
it looks like this is always false?

Reply:
Ah, yeah, I forgot I'd removed that path because I was able to thread the delayed persistent assets after the block in LoadPersistenAssets.